File: src/dotnet/ArdsTools/Ards.Tools.Domain/Models/ArrlSection.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Ards.Tools.Domain.Adif.Adif.Models
{   
    /// <summary>
    /// ARRL Section Enumeration
    /// </summary>
    [Table("arrl_section", Schema = "adif")]
    public partial class ArrlSection
    {
        [Column("id")]
        public int Id { get; set; }
        [Required]
        [Column("abbreviation")]
        [StringLength(4)]
        public string Abbreviation { get; set; }
        [Required]
        [Column("name")]
        [StringLength(70)]
        public string Name { get; set; }
        [Column("dxcc_id")]
        public int DxccId { get; set; }
        [Column("from_date", TypeName = "date")]
        public DateTime? FromDate { get; set; }
        [Column("deleted_date", TypeName = "date")]
        public DateTime? DeletedDate { get; set; }

        [ForeignKey("DxccId")]
        [InverseProperty("ArrlSection")]
        public virtual Dxcc Dxcc { get; set; }
    }
}

File: src/java/ards-adif-api/src/main/java/com/ards/adif/api/model/ArrlSection.java
package com.ards.adif.api.model;

import java.io.Serializable;
import javax.persistence.*;
import java.util.Date;


/**
 * The persistent class for the arrl_section database table.
 * 
 */
@Entity
@Table(name="arrl_section")
@NamedQuery(name="ArrlSection.findAll", query="SELECT a FROM ArrlSection a")
public class ArrlSection implements Serializable {
	private static final long serialVersionUID = 1L;

	@Id
	@Column(unique=true, nullable=false)
	private Integer id;

	@Column(nullable=false, length=4)
	private String abbreviation;

	@Temporal(TemporalType.DATE)
	@Column(name="deleted_date")
	private Date deletedDate;

	@Temporal(TemporalType.DATE)
	@Column(name="from_date")
	private Date fromDate;

	@Column(nullable=false, length=70)
	private String name;

	//bi-directional many-to-one association to Dxcc
	@ManyToOne
	@JoinColumn(name="dxcc_id", nullable=false)
	private Dxcc dxcc;

	public ArrlSection() {
	}

	public Integer getId() {
		return this.id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public String getAbbreviation() {
		return this.abbreviation;
	}

	public void setAbbreviation(String abbreviation) {
		this.abbreviation = abbreviation;
	}

	public Date getDeletedDate() {
		return this.deletedDate;
	}

	public void setDeletedDate(Date deletedDate) {
		this.deletedDate = deletedDate;
	}

	public Date getFromDate() {
		return this.fromDate;
	}

	public void setFromDate(Date fromDate) {
		this.fromDate = fromDate;
	}

	public String getName() {
		return this.name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public Dxcc getDxcc() {
		return this.dxcc;
	}

	public void setDxcc(Dxcc dxcc) {
		this.dxcc = dxcc;
	}

}